assets/js/bell.js

```diff
@@ -18,29 +18,29 @@ class View {
       this.waitingClients.push(clientId)
     }
 
-    this.render();
+    this.render()
   }
 }
 function getBellId() {
-  return Maybe(document.getElementById("bell"))
+  return Maybe(document.getElementById('bell'))
     .getAttribute('data-bell-id')
-    .value;
+    .value
 }
 
 export default function(socket) {
-  const bellId = getBellId();
-  if (!bellId) return;
+  const bellId = getBellId()
+  if (!bellId) return
 
   const client = new Client(bellId)
   const view = new View(client)
 
   const channel = socket.channel(`door:${bellId}`, {})
 
   channel.join()
-    .receive("ok", resp => { console.log("Joined the channel") })
-    .receive("error", resp => { console.log("Unable to join", resp) })
+    .receive('ok', resp => { console.log('Joined the channel') })
+    .receive('error', resp => { console.log('Unable to join', resp) })
```

Comment:
Unexpected console statement.

[Read more about it here.](http://eslint.org/docs/rules/no-console)